src/pkg/services/m365/m365_test.go

```diff
@@ -0,0 +1,62 @@
+package m365
+
+import (
+	"testing"
+
+	"github.com/alcionai/clues"
+	"github.com/alcionai/corso/src/internal/tester"
+	"github.com/alcionai/corso/src/pkg/account"
+	"github.com/alcionai/corso/src/pkg/credentials"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type M365IntgSuite struct {
+	tester.Suite
+}
+
+func TestM365IntgSuite(t *testing.T) {
+	suite.Run(t, &M365IntgSuite{
+		Suite: tester.NewIntegrationSuite(
+			t,
+			[][]string{}),
+	})
+}
+
+func (suite *userIntegrationSuite) TestNewM365Client_invalidCredentials() {
+	table := []struct {
+		name string
+		acct func(t *testing.T) account.Account
+	}{
+		{
+			name: "Invalid Credentials",
+			acct: func(t *testing.T) account.Account {
```

Comment:
there's actually a `tester.NewFakeM365Account` function you can use now if you'd be interested

Reply:
Heyo!  Sure am interested.